### PR TITLE
Fix SciPyDistribution range computation.

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by regislebrun** [comment]
@jschueller  You should use the relevant ResourceMap key instead of an hard-coded 1e-12 to be consistent with the C++ library and to allow the user to change this value if needed. I don't know why I get an error when I try to add a single comment in the relevant part of the code on github...

File: python/src/Distribution.i [hunk 1]
```diff
@@ -51,8 +60,9 @@ class SciPyDistribution(PythonDistribution):
         self._dist = dist
 
         # compute range
-        lb = dist.ppf(0.)
-        ub = dist.ppf(1.)
+        cdf_epsilon = 1e-12
```
Comment (regislebrun) [inline python/src/Distribution.i]: the computation should be done first using 0.0 and 1.0, and only the infinite bounds should be replaced by the value at 1.0e-12 or 1-1.0e-12. This way, you get the exact value for the finite bounds and a useful value for the infinite bounds

**Comment by regislebrun** [comment]
@jschueller There is also a small bug to fix: scipy computes the *excess* kurtosis while OT computes the *exact* kurtosis. It means that we have to add 3.0 (=kurtosis of Normal()) to the value given by scipy in order to get the value expected by OT

**Comment by regislebrun** [comment]
I wrote a small benchmark comparing OT distributions, scipy distributions and wrapped scipy distributions. I used the standard normal distribution and the johnsonsu distribution. The results and the code are given below. The main observations are:
- pure OT distributions are approximately as efficient as their scipy conterparts. I would have expected a better result for OT due to parallelism and C++ vs Python speed. Something has to be improved here.
- the penalty paid by encapsulating a scipy distribution is huge, in particular regarding the quantile computation. It is a bad news for those who want to use the associated isoprobabilistic transformation (PCE, FORM etc) where the speed of the transformation can take a significant amount of the total time.
```
import openturns as ot
import scipy.stats as st
from time import time

lamb = 1.5
xi = 1.1
delta = 2.0
gamma = 1.0
OT_dist = [((ot.Normal() - gamma) / delta).sinh() * lamb + xi, ot.Normal(0.0, 1.0)]
ST_dist = [st.johnsonsu(gamma, delta, loc=xi, scale=lamb), st.norm()]
l_size = [1000, 10000]
# Bench
for i in range(len(OT_dist)):
    size = l_size[i]
    u = ot.RandomGenerator.Generate(size)
    distribution = OT_dist[i]
    print("\n"+"*"*50)
    print(distribution.getClassName())
    print("*"*50)
    print("OT distribution")
    t0 = time()
    sample = distribution.getSample(size)
    print("sampling speed=%.4e" % (size / (time() - t0)), "rng/s")
    t0 = time()
    pdf = distribution.computePDF(sample)
    print("pdf speed=%.4e" % (size / (time() - t0)), "pdf/s")
    t0 = time()
    cdf = distribution.computeCDF(sample)
    print("cdf speed=%.4e" % (size / (time() - t0)), "cdf/s")
    t0 = time()
    quantile = distribution.computeQuantile(u)
    print("quantile speed=%.4e" % (size / (time() - t0)), "quantile/s")
    print("#"*50)

    print("Pure scipy")
    scipyJU = ST_dist[i]
    t0 = time()
    sample = scipyJU.rvs(size)
    print("sampling speed=%.4e" % (size / (time() - t0)), "rng/s")
    t0 = time()
    pdf = scipyJU.pdf(sample)
    print("pdf speed=%.4e" % (size / (time() - t0)), "pdf/s")
    t0 = time()
    cdf = scipyJU.pdf(sample)
    print("cdf speed=%.4e" % (size / (time() - t0)), "cdf/s")
    t0 = time()
    quantile = scipyJU.ppf(u)
    print("quantile speed=%.4e" % (size / (time() - t0)), "quantile/s")
    print("#"*50)

    pyDist = ot.SciPyDistribution(scipyJU)
    distribution = ot.Distribution(pyDist)
    print("Wrapped scipy")
    t0 = time()
    sample = distribution.getSample(size)
    print("sampling speed=%.4e" % (size / (time() - t0)), "rng/s")
    t0 = time()
    pdf = distribution.computePDF(sample)
    print("pdf speed=%.4e" % (size / (time() - t0)), "pdf/s")
    t0 = time()
    cdf = distribution.computeCDF(sample)
    print("cdf speed=%.4e" % (size / (time() - t0)), "cdf/s")
    t0 = time()
    quantile = distribution.computeQuantile(u)
    print("quantile speed=%.4e" % (size / (time() - t0)), "quantile/s")
```
which gives:
```
**************************************************
RandomMixture
**************************************************
OT distribution
sampling speed=1.3870e+06 rng/s
pdf speed=2.4770e+05 pdf/s
cdf speed=3.6708e+05 cdf/s
quantile speed=2.2989e+04 quantile/s
##################################################
Pure scipy
sampling speed=3.5636e+06 rng/s
pdf speed=3.0728e+06 pdf/s
cdf speed=3.9984e+06 cdf/s
quantile speed=5.9000e+05 quantile/s
##################################################
Wrapped scipy
sampling speed=6.6062e+05 rng/s
pdf speed=9.2139e+03 pdf/s
cdf speed=1.1398e+04 cdf/s
quantile speed=9.0660e+02 quantile/s

**************************************************
Normal
**************************************************
OT distribution
sampling speed=2.1109e+07 rng/s
pdf speed=1.3565e+07 pdf/s
cdf speed=1.2112e+07 cdf/s
quantile speed=1.3194e+07 quantile/s
##################################################
Pure scipy
sampling speed=1.7127e+07 rng/s
pdf speed=1.5044e+07 pdf/s
cdf speed=1.5552e+07 cdf/s
quantile speed=7.2552e+05 quantile/s
##################################################
Wrapped scipy
sampling speed=7.2486e+05 rng/s
pdf speed=1.3126e+04 pdf/s
cdf speed=1.5839e+04 cdf/s
quantile speed=1.5397e+03 quantile/s
```

**Comment by jschueller** [comment]
@regislebrun I just fixed these issues

**Comment by regislebrun** [comment]
+1 Phil will be happy!

**Pull request merged**